rust/libnewsboat/src/utils.rs

```diff
@@ -634,23 +634,6 @@ pub fn mkdir_parents<R: AsRef<Path>>(p: &R, mode: u32) -> io::Result<()> {
         .create(p.as_ref())
 }
 
-/// Extracts up to `n` first graphemes from the given string.
-///
-/// ```
-/// use libnewsboat::utils::take_graphemes;
-///
-/// let input = "Привет!";
-/// assert_eq!(take_graphemes(input, 1), "П");
-/// assert_eq!(take_graphemes(input, 4), "Прив");
-/// assert_eq!(take_graphemes(input, 6), "Привет");
-/// assert_eq!(take_graphemes(input, 20), input);
-/// ```
-pub fn take_graphemes(input: &str, n: usize) -> String {
-    UnicodeSegmentation::graphemes(input, true)
```

Comment:
[rustc warns](https://cirrus-ci.com/task/5940998089998336?command=build#L338) that this was the last use of `UnicodeSegmentation`, and a quick `git grep` shows that we don't need `unicode_segmentation` crate anymore, so it can be removed from rust/libnewsboat/Cargo.toml now. (I should've probably ask about all of this is my previous comment, sorry about that)

Reply:
No problem, just fixed it.
I just find it hard to see the relevant warnings given the amount of `deprecated trim_end_matches` warnings.

Reply:
299c311 removes the import, but not the crate?

Reply:
> I just find it hard to see the relevant warnings given the amount of deprecated trim_end_matches warnings.

Yeah, everybody is tired of them by now, so after 2.19 release we'll switch to a more aggressive schedule regarding Rust (so we don't have to look at warnings for long), and I'll probably enable `-Werror` like we do for C++.

Reply:
Apparently the crate uses a dash where the `extern crate` uses an underscore.
That's why my `git grep unicode_segmentation` didn't find the crate usage in the Cargo.toml file (and I somehow overlooked the actual filename you mentioned, sorry about that).